examples/app/weather/templates/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{ title if title else "Bokeh Plot" }}</title>
+        {{ bokeh_css }}
+        {{ bokeh_js }}
+    </head>
+    <body>
+        <div>
+            HERE IS SOME CUSTOM STUFF
+        </div>
+        {{ plot_div|indent(8) }}
+        {{ plot_script|indent(8) }}
```

Comment:
other than clarity, making this less granular would mean fewer apps break if the implementation details ever change...